kubernetes/assets/monitors/monitor_deployments_replicas.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "[kubernetes] Monitor Kubernetes Deployments Replica Pods",
 	"type": "query alert",
-	"query": "avg(last_15m):avg:kubernetes_state.deployment.replicas_desired{*} by {kube_namespace,kube_deployment} - avg:kubernetes_state.deployment.replicas_available{*} by {kube_namespace,kube_deployment} >= 2",
+	"query": "avg(last_15m):avg:kubernetes_state.deployment.replicas_desired{*} by {kube_cluster_name,kube_namespace,kube_deployment} - avg:kubernetes_state.deployment.replicas_available{*} by {kube_cluster_name,kube_namespace,kube_deployment} >= 2",
```

Comment:
How does it behaves if the `kube_cluster_name` is not present? As it's an optional tag, it is enforced on our side to enable the orchestrator explorer feature but it's possible that the tag is missing

https://github.com/DataDog/datadog-agent/blob/b68228d941027d4255dd225f0223926ff6c6bae1/pkg/metadata/host/host_tags.go#L84-L91

Reply:
It will be N/A but that should be fine 🤔